src/components/ContentNode.vue

```diff
@@ -38,6 +39,7 @@ const InlineType = {
   codeVoice: 'codeVoice',
   emphasis: 'emphasis',
   image: 'image',
+  video: 'video',
```

Comment:
Is a video really an "inline" element? I imagine we would be rendering all videos as distinct blocks of content, but I could be mistaken.

Reply:
You do have a point. I just followed the InlineImage.vue component, but that probably is not correct here.

Reply:
@ethan-kusters what do you think? Should the video be `inline` like the image, even though our images are not really inline either... :D I wont hurry to change anything for now.

Reply:
Hm... I think we should leave it as inline even though I agree we're kind of misusing that term. We want video supported anywhere image is and we currently consider those inline.

We could consider refactoring all of those in the future but I think it's a separate task.

<img width="541" alt="Screenshot 2022-09-09 at 12 22 54 PM" src="https://user-images.githubusercontent.com/6750147/189339518-d2fa153b-00fe-48bb-958f-6aaf72eb077b.png">

Reply:
I chatted with @franklinsch about this and came to the opposite conclusion. 😁 

Commonmark markdown actually supports inline images which is why we use an inline element for it even though, today, DocC always wrap images in paragraphs which effectively makes them block elements. In the future we might want to actually properly support inline images so it makes sense to leave this as-is.

However, since DocC videos can only be authored with directives and we don't support inline directives (and we don't see a use-case for inline videos) it doesn't make sense for videos to be inline elements. I'll put up a PR momentarily but I'm updating the spec to make Video a block element based on this- let me know if ya'll disagree with this general line of logic though. Happy to discuss further.

Reply:
Spec update for this is here: https://github.com/apple/swift-docc/pull/381/files#diff-fc7c8431152d3d494899ec4b551f1a5d1249ed44c853e05da1cc26374d9d4c54.

Reply:
Sounds reasonable to me. The block vs inline image thing will be a pain in the future in terms of backwards compatibility but that's entirely separate from this.